inspirehep/dojson/hep/fields/bd76x78x.py

```diff
@@ -94,22 +95,23 @@ def publication_info2marc(self, key, value):
         page_artid.append('{page_start}'.format(**value))
     if value.get('artid'):
         page_artid.append('{artid}'.format(**value))
-    return {
-        '0': get_recid_from_ref(
-            value.get('parent_record')),
-        'c': page_artid,
-        'n': value.get('journal_issue'),
-        'o': value.get('conf_acronym'),
-        'p': value.get('journal_title'),
-        'r': value.get('reportnumber'),
-        't': value.get('confpaper_info'),
-        'v': value.get('journal_volume'),
-        'w': value.get('cnum'),
-        'x': value.get('pubinfo_freetext'),
-        'y': value.get('year'),
-        'z': value.get('isbn'),
-        'm': value.get('note')
-    }
+    ret = {'0': get_recid_from_ref(value.get('parent_record')),
+           'c': page_artid,
+           'n': value.get('journal_issue'),
+           'o': value.get('conf_acronym'),
+           'p': value.get('journal_title'),
+           'r': value.get('reportnumber'),
+           't': value.get('confpaper_info'),
+           'v': value.get('journal_volume'),
+           'w': value.get('cnum'),
+           'x': value.get('pubinfo_freetext'),
+           'y': value.get('year'),
+           'z': value.get('isbn'),
+           'm': value.get('note')}
+    if value.get('hidden'):
+        self.setdefault('7731_', []).append(ret)
```

Comment:
Uh, let me understand the use case here, because this is hidden, but it's still part of the metadata, just under a different key...